crates/torii/server/src/cli.rs

```diff
@@ -154,3 +121,73 @@ async fn main() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+// Tries to find scarb manifest first for env variables
+//
+// Use manifest path from cli args,
+// else uses scarb manifest to derive path of dojo manifest file,
+// else try to derive manifest path from scarb manifest
+// else try `./target/dev/manifest.json` as dojo manifest path
+//
+// If neither of this work return an error and exit
+fn get_manifest_and_env(
+    args_path: Option<&Utf8PathBuf>,
+) -> anyhow::Result<(Manifest, Option<Environment>)> {
+    // If there is scarb manifest file it shoul
+    let config;
```

Comment:
this var doesn't seem to be used anywhere else apart from the `if` branch below.